salt/returners/slack_returner.py

```diff
@@ -231,4 +241,4 @@ def returner(ret):
                           username,
                           as_user,
                           api_key)
-    return slack
+    return slack
```

Comment:
Please add a newline below this.

Reply:
@cachedout Sorry I didnt notice it got deleted. Added back in.